modules/getting-started-cli-granting-permissions.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-cli.adoc
+
+[id="getting-started-cli-granting-permissions_{context}"]
+= Granting view permissions
+
+{product-title} automatically creates a few special service accounts in every project. The default service account takes responsibility for running the Pods. {product-title} uses and injects this service account into every Pod that launches.
+
+The following procedure creates a Service Account named `sa-user-account`. The service account communicates with the {product-title} API to learn about Pods, Services, and resources within the project.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+* You have a deployed image.
+* You must have `cluster-admin` privileges.
```

Comment:
project admin user is also able to grant other user view permission, `cluster-admin` is NOT a must